src/components/App.js

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react';
+import '../styles/app.css';
+import Search from '../components/Search';
+import SearchResults from './SearchResults';
+
+const App = () => {
+  const [searchResults, setSearchResults] = useState([]);
+  console.log(searchResults);
+
+  return (
+    <div className="app">
+      <img className="nasaImage" 
+      src="https://cdn.cnn.com/cnnnext/dam/assets/200424060716-nasa-worm-logo.jpg" 
+      alt="nasaLogo"/>
```

Comment:
This should be indented by another level:
```suggestion
      <img
        className="nasaImage"
        src="https://cdn.cnn.com/cnnnext/dam/assets/200424060716-nasa-worm-logo.jpg"
        alt="nasaLogo"
      />
```

Also, the `alt` tag could be a bit more user friendly here. Its designed for screen readers so you can put short descriptions here that would be for more use, eg 'Large NASA logo in bold red text'